examples/optuna/README.md

```diff
@@ -0,0 +1,32 @@
+# HYPERPARAMETER OPTIMIZATION WITH OPTUNA 
+
+## Requirements
+
+- Optuna
+
+## Description
+
+chainer_simple.py
```

Comment:
How about linking to example codes in Optuna repository from this README, instead of coping  them to Chainer repository? I think it's better in terms of future maintenance by Optuna team.
```suggestion
[chainer_simple.py](https://github.com/optuna/optuna/blob/master/examples/chainer_simple.py)
```

Reply:
@Crissman How do you think?